qmd/_quarto.yml

```diff
@@ -9,20 +9,26 @@ book:
   date-format: iso
   license: CC 4.0 BY-SA
   repo-url: "https://github.com/Big-Life-Lab/PHES-ODM-Doc"
+  repo-actions: [edit, issue, source]
```

Comment:
I don't think we should be including the edit option.

Reply:
removed.